src/index.html

```diff
@@ -3,11 +3,240 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Miami</title>
+    <title>MyBiKE</title>
     <link rel="stylesheet" href="./styles/main.scss">
     <script src="scripts/main.js" defer></script>
   </head>
-  <body>
-    <h1>Miami</h1>
+  <body class="page">
+    <div class="container">
+      <header class="header">
+        <img
+          src="./images/MyBiKE.svg"
+          alt="My bike logo"
+          class="header__img"
+        >
+
+        <img
+          src="./images/phone.svg"
+          alt="call us"
+          class="header__img"
+        >
+
+        <img
+          src="./images/menu.svg"
+          alt="menu"
+          class="header__img"
+        >
+
+        <h1 class="header__h1">
+          Take the Streets
+        </h1>
+      </header>
+
+      <section class="tagline">
+        <h2 class="tagline__h2">
+          Move Free
+        </h2>
+        <p class="p--special">
+          Shift your ride, not gears. Find the fastest way to move in the city as the bike adapts intuitively to power the speed you need.
+        </p>
+      </section>
+
+      <section class="compare-bikes">
+        <h2>Compare Bikes</h2>
+
+        <div class="bike">
```

Comment:
I did it intentionally, because I liked Yura's approach regarding this. because the bike card is a separate block that can be used not only in this particular section.

Reply:
like product card

Reply:
let me know, pls, if in ur point of view it`s necessary and certainly bug, not feature

Reply:
In this case, could leave as you already have(for bikes).

Reply:
Ok, thanks, done. Did u get my rework? Cos I not sure if it upload

Reply:
But better rename and in this case you could reuse all section. Because ofter you could see at you site `nav, logo, title` - almost the same styles. But not one bike from this section, all section - possible.